lib/util/template/index.ts

```diff
@@ -56,6 +56,7 @@ export const exposedConfigOptions = [
   'commitMessagePrefix',
   'commitMessageSuffix',
   'commitMessageTopic',
+  'extends',
```

Comment:
This does not achieve what you want. It exposes extends which is not desired

Reply:
Thx, this looked way too easy. Now I got the meaning of that Array.